docs/app/Examples/modules/Checkbox/Variations/Variations.js

```diff
@@ -10,7 +11,15 @@ export default class CheckboxVariationsExamples extends Component {
           title='Fitted'
           description='A fitted checkbox does not leave padding for a label'
           examplePath='modules/Checkbox/Variations/Fitted'
-        />
+        >
+          <Message>
+            The&nbsp;
+            <a href='http://semantic-ui.com/modules/checkbox.html#fitted' target='_blank'>
+              <code>fitted</code>
+            </a>
+            &nbsp;class is automatically applied if there is no <code>label</code> prop.
```

Comment:
Yes, React removes all whitespace regardless of how many blank lines (unlike html).  So, there is no space between words/components without this.